protos/guard/guard.proto

```diff
@@ -123,6 +125,17 @@ message Contract {
   int32 challenge_success_times = 16; //record times of success after the contract was put into the warn status
 }
 
+message FileInfoResponse {
+  repeated FileStoreMeta file_store_meta = 1 [
```

Comment:
Suggest to use repeated string file_hashes here, as file_store_meta have too many information, retrieve it in one package will cost more bandwidth.

Reply:
We need return file_hash, file size, rent start and rent end for the response, FileStoreMeta contains more attributes, but seems they will not take up big bandwidth.

Reply:
Maybe you need list of `Contract`, same as the `rpc ListHostContracts(ListHostContractsRequest) returns (ContractsList);` endpoint. It will return everything you need.

Reply:
Return FileStoreMeta list and file count information.